app/_data/docs_nav_konnect.yml

```diff
@@ -60,7 +60,15 @@
     - text: Overview
       url: /runtime-manager
     - text: Manage a Kong Gateway Runtime
-      url: /runtime-manager/kong-gateway-runtime
+      items:
+        - text: Docker
+          url: /runtime-manager/gateway-runtime-docker
+        - text: Kubernetes
+          url: /runtime-manager/gateway-runtime-kubernetes
+        - text: Universal (kong.conf)
```

Comment:
Do we use this Universal label elsewhere in the docs? Should this just be 'Linux'?

Reply:
Only in the Mesh docs. Linux works!